### PR TITLE
onboarding and collet page ui for hot of flame

### PR DESCRIPTION
**Pull request by shahhardik4599**

### CODE REVIEW

File: components/Card/Card.module.scss [hunk 1]
```diff
@@ -123,6 +123,7 @@
   place-content: end;
   position: relative;
   overflow: hidden;
+
```
Comment (rachinkapoor) [inline components/Card/Card.module.scss:126]: Discard change.

File: components/Card/index.js [hunk 2]
```diff
@@ -124,72 +131,92 @@ export default function Card({ cardDetails, showCollectModal }) {
...
-              ? "You have already collected this"
-              : "Collect Now"}
+          <span className="font-medium text-[16px] leading-[26px] text-[#ffffff] ml-[3px]">
+            {cardDetails.totalVotes}
           </span>
-        </button>
+        </div>
       </div>
-    </div>
+    </>
```
Comment (rachinkapoor) [inline components/Card/index.js:220]: remove this tag, not needed.

File: components/CollectNFT/SubComponent/CollectNFT.js [hunk 3]
```diff
@@ -75,6 +75,7 @@ function CollectNFT(props) {
             hasCollected:
               !!currentUserLensPostRelation?.collect_nft_transaction_hash,
             handle: ownerUser?.lens_profile_username,
+            totalVotes: lensPostDetail?.total_votes,
```
Comment (rachinkapoor) [inline components/CollectNFT/SubComponent/CollectNFT.js:78]: ok for now, next time try avoiding creating custom entities or data.

File: components/CollectNFT/SubComponent/SVG/leftArrow.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ function LeftArrow(props) {
         <path
           d="M28.75 36.4167L15.3333 23L28.75 9.58334"
           stroke="white"
-          strokeOpacity="0.8"
+          strokeOpacity={hovered ? "1" : "0.8"}
```
Comment (rachinkapoor) [inline components/CollectNFT/SubComponent/SVG/leftArrow.js:16]: Nop, don't use state. Change to css.

File: components/CollectNFT/SubComponent/SVG/rightArrow.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-function RightArrow(props) {
+function RightArrow({ hovered }) {
```
Comment (rachinkapoor) [inline components/CollectNFT/SubComponent/SVG/rightArrow.js:3]: dont use state, use CSS

File: components/CollectNFT/SubComponent/hallOfFlameModal.js [hunk 6]
```diff
@@ -17,6 +17,7 @@ function HallOfFlameModal({
   onCollectNow,
 }) {
   const Themes = TRENDING_THEMES;
+  const [hovered, setHovered] = React.useState(false);
```
Comment (rachinkapoor) [inline components/CollectNFT/SubComponent/hallOfFlameModal.js:20]: Dont use state, use CSS

File: components/OnBoarding/OnBoarding.module.scss [hunk 7]
```diff
@@ -1,3 +1,103 @@
...
+
+  .voteCard {
+    &:last-child {
+      // transform: rotate(0deg) scale(1) !important;
+      transition-delay: 0.25s;
+      transition-duration: 1s;
+    }
+
+    &:nth-last-child(2) {
+      transform: rotate(5deg) scale(0.92) !important;
```
Comment (rachinkapoor) [inline components/OnBoarding/OnBoarding.module.scss:48]: Fix the animation.

File: components/OnBoarding/OnBoardingContent.js [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+                {onBoardingDetailsArray.type === "gif" ? (
+                  <Image
+                    src={onBoardingDetailsArray.gif}
+                    width={155}
+                    height={210}
+                    alt="my gif"
+                    className="max-w-[150%]"
+                  />
+                ) : (
+                  onBoardingDetailsArray.gif
```
Comment (rachinkapoor) [inline components/OnBoarding/OnBoardingContent.js:46]: Never ever, ever ever, EVER! commit images into code base.

File: components/TopBar/SignInButton/index.js [hunk 9]
```diff
@@ -248,7 +248,7 @@ export default function SignInButton({
             onClose={() => setShouldShowEnableDispatcherModal(false)}
           />
         ) : null}
-        <AboutLens isOpen={isSignInModalOpen} />
+        {/* <AboutLens isOpen={isSignInModalOpen} /> */}
```
Comment (rachinkapoor) [inline components/TopBar/SignInButton/index.js:251]: Dont comment, delete.

File: components/vote/svg/FirstTimeUser/FireSmallSvg.js [hunk 10]
```diff
@@ -0,0 +1,174 @@
+import * as React from "react";
+
+const FireSmallSvg = (props) => (
```
Comment (rachinkapoor) [inline components/vote/svg/FirstTimeUser/FireSmallSvg.js:3]: Fix SVG errors.

**Review by rachinkapoor — CHANGES_REQUESTED** [review]

File: components/OnBoarding/OnBoardingContent.js (hunk 8 above)
Comment (shahhardik4599) [reply components/OnBoarding/OnBoardingContent.js:46]: Got it sir, will never do that

File: components/OnBoarding/OnBoarding.module.scss (hunk 7 above)
Comment (shahhardik4599) [reply components/OnBoarding/OnBoarding.module.scss:48]: animation is still pending

**Review by rachinkapoor — APPROVED** [review]
Apart from animation fixes, looks good.

**Pull request merged**